src/js/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -92,7 +109,33 @@ class VeteranBenefitSummaryLetter extends React.Component {
           recent periods of service.
         </p>
         <h2>Military Service Information</h2>
-        {militaryServiceInfo}
+        <div className="form-checkbox">
+          <input
+              autoComplete="false"
+              id="serviceInfoCheckboxId"
```

Comment:
Awesome, you'll want to pull down the branch before you make that change, because I just rebased this on master :/ sorry about that, wanted to update it before merging